caddyhttp/httpserver/context.go

```diff
@@ -57,6 +58,24 @@ func (c Context) Header(name string) string {
 	return c.Req.Header.Get(name)
 }
 
+// Env gets a map of the environment variables.
+func (c Context) Env() map[string]string {
```

Comment:
How about instead of getting all of them we just accept an argument which is the env var name and then return its value? Then it's one line: os.GetEnv(). I'd like that much better...